<launch>
  <arg name="debug" default="false" />
  <arg unless="$(arg debug)" name="launch_prefix" value="" />
  <arg     if="$(arg debug)" name="launch_prefix"
	   value="/usr/local/bin/gdb --directory=/usr/local/bin --ex run --args" />
  <arg name="octomap_topic" default="/camera/depth/points"/>
  <arg name="hist_resolution" default="0.05" />
  <arg name="max_plan_range" default="1.0" />
  <arg name="map_resolution" default="0.15" />

  <!-- Start the mavros navigation node -->
  <node name="vfh3d_node" pkg="vfh3d" type="vfh3d_node" output="screen" launch-prefix="$(arg launch_prefix)">
    <param name="octomap_topic" value="$(arg octomap_topic)" />
    <param name="hist_resolution" value="$(arg hist_resolution)" />
    <param name="max_plan_range" value="$(arg max_plan_range)" />
    <param name="map_resolution" value="$(arg map_resolution)" />
  </node>
</launch>
